.github/workflows/memote-history.yml

```
@@ -35,11 +35,18 @@ jobs:
         ref: ${{ steps.setup.outputs.deployment }}
         path: ${{ steps.setup.outputs.other-repo }}
 
+    - name: Convert model
+      run: |
```

You can consider the following so you don't need to escape line breaks.
```suggestion
      run: >
```

---

Good suggestion! I would rather keep the line breaks for easier copy-pasting into different shell, although I think `zsh` wouldn't need these. @BenjaSanchez  any preference for `|` or `>`?

---

I'm afraid I don't understand the implications of either, could you ellaborate?

---

Sorry, I was not very clear. `: >` means you can break the lines as you please and each newline will be transformed into a space. So you can break up a command over multiple lines without escaping them. `: |` on the other hand considers every line separately. This is useful for separate commands or you need to escape line breaks as you have done here with `\`.

You can read more about it in the spec https://yaml.org/spec/1.2/spec.html#id2760844

---

@Midnighter thanks for the explanation. @mihai-sysbio `>` indeed seems simpler so I would use that one